versions.yaml

```diff
@@ -155,7 +155,7 @@ assets:
   kernel:
     description: "Linux kernel optimised for virtual machines"
     url: "https://cdn.kernel.org/pub/linux/kernel/v5.x/"
-    version: "v5.10.25"
+    version: "v5.15.17"
```

Comment:
According to https://www.kernel.org/ , the LTS version was bumped to .18 two days ago:
```
longterm:     5.15.18     2022-01-29
```
Maybe worth doing the same in kata ?

Reply:
Updated to `.19` :)